frontend/src/components/TopNavigation.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <header class="static flex flex-row items-center justify-between px-20 py-6 pb-8 border-b">
-    <h1 class="py-2 font-medium cursor-pointer text-2xl" @click="$router.push('/').catch((err) => {})">📦</h1>
+  <header class="static flex flex-row items-center justify-between px-16 py-2 md:py-3 border-b border-primary shadow-md bg-moderngrey bg-gradient-to-t from-gray">
```

Comment:
Padding is still too large on mobile

![image](https://user-images.githubusercontent.com/36769448/122059214-afc79280-cdec-11eb-9f98-1b1983ed04f7.png)